### PR TITLE
feat: a script to remove selected bills

### PR DESCRIPTION
**Pull request by doubleface**
By default, this script does nothing. You have to modify it to select the bills you want to delete
There may/should be a better/cleaner way to do this but deleting documents should not be too easy I think

### CODE REVIEW

File: scripts/removeBills.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+const DOCTYPE_BILLS = 'io.cozy.bills'
+const get = require('lodash/get')
+
+// define your bills filter here
+// I am sure there is a better wait to configure that in ACH
+//
+function billsToRemoveFilter(bill) {
+  // return bill.vendor === 'impot'
+  // return get(bills.cozyMetadata.createdByApp) === 'impot' && get(bills.cozyMetadata.createdByAppVersion === "1.9.0")
```
Comment (ptbrowne) [inline scripts/removeBills.js]: Would be interesting to do that directly with a mango selector.

```
ACH script removeDocuments '{ cozyMetadata: { createdByApp: 'impot', createdByAppVersion: '1.9.0' } }'
```

[sift](https://www.npmjs.com/package/sift) can be useful for this.

File: scripts/removeBills.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    console.log('Deleting bills...')
+    await api.deleteAll(DOCTYPE_BILLS, billsToRemove)
+    console.log('  done')
+  }
+}
+
+module.exports = {
+  getDoctypes: () => [DOCTYPE_BILLS],
+  run: async function(ach, dryRun = true) {
+    return run(mkAPI(ach.client), dryRun).catch(err => {
```
Comment (ptbrowne) [inline scripts/removeBills.js]: The `run` function signature has changed recently.

It is `run(context)` and context contains:

- client: a *new* cozy-client
- dryRun
- stats: a shared object used to count thing

**Comment by ptbrowne** [comment]
>  There may/should be a better/cleaner way to do this but deleting documents should not be too easy I think

IMHO changing a file is not too difficult but it is annoying. To me what is important is that the person doing the task can quickly assess what is going to happen, and that it should be hard to make mistakes.

- Here the dryRun behavior is already implemented and makes it easy to assess the behavior
- For the making mistakes part, I think we should have an auto backup of the documents that are updated, this way it would be easy to rollback.

**Comment by nono** [comment]
What is the status of this PR?

**Comment by nono** [comment]
Ping @doubleface

**Pull request closed without merging**